### PR TITLE
hxRaylib overhaul - Build using a bat file, updated README.md, code, and more.

### PR DESCRIPTION
**Pull request by polybiusproxy**
Title says all, a full README.md edit, made a bat file for easier builds, updated example code a bit, and added a gitignore.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,18 +4,26 @@ Haxe bindings for Raylib, a simple and easy-to-use library to learn videogame pr
 ###### warning: the binding is in its early stage of development so use at your own risk
 
 # Requirements
-- Haxe
-- hxcpp
-- Visual studio 2019 or build tools for vs2019
+
+- Haxe 4.0.0 of above.
```
Comment (logo4poop) [inline README.md:8]:
```suggestion
- Haxe 4.0.0 or above.
```

File: README.md [hunk 2]
```diff
@@ -4,18 +4,26 @@ Haxe bindings for Raylib, a simple and easy-to-use library to learn videogame pr
...
+- hxcpp (Haxe's C++ runtime) - Install it by running `haxelib install hxcpp`.
+- Raylib 3.7.0 source code. - Get it from [here](https://github.com/raysan5/raylib/archive/refs/tags/3.7.0.zip).
+
+- The following C++ tools: (you can find them in the VS Installer)
```
Comment (logo4poop) [inline README.md:12]:
```suggestion
- The following C++ tools for windows: (you can find them in the VS Installer)
```

File: README.md [hunk 3]
```diff
@@ -4,18 +4,26 @@ Haxe bindings for Raylib, a simple and easy-to-use library to learn videogame pr
...
+    * Windows SDK (latest version)
 
 # Installation
-- Clone this repository
-- You also need the raylib source files in the format `lib -> raylib -> src`
-- This repo can be used as a template 
--  To compile: `haxe -cp src -cpp your output dir -main Main`
+
+1. Clone this repository.
+2. Create a folder called "lib".
```
Comment (logo4poop) [inline README.md:19]:
```suggestion
2. Create a folder called "lib" in the root direcoty.
```

File: README.md [hunk 4]
```diff
@@ -4,18 +4,26 @@ Haxe bindings for Raylib, a simple and easy-to-use library to learn videogame pr
...
+3. Rename the "raylib-3.7.0" folder to just "raylib".
+4. Insert the "raylib" folder into the "lib" folder. Then path should be like this: `lib -> raylib -> src`
```
Comment (logo4poop) [inline README.md:21]:
```suggestion
4. Insert the "raylib" folder into the "lib" folder. Then path should be like this: `lib/raylib/src`
```

File: README.md [hunk 5]
```diff
@@ -4,18 +4,26 @@ Haxe bindings for Raylib, a simple and easy-to-use library to learn videogame pr
...
+5. To compile, run hxBuild.bat
```
Comment (logo4poop) [inline README.md:22]:
```suggestion
5. To compile for windows, run hxBuild.bat
```

File: hxBuild.bat [hunk 6]
```diff
@@ -0,0 +1,51 @@
+@echo off
+cls
+TITLE hxRaylib - Building...
+CALL :setESC
+GOTO START
+
+:START:
+ECHO %ESC%[45mhxRaylib builder - Made by PolybiusProxy%ESC%[0m
```
Comment (logo4poop) [inline hxBuild.bat:8]:
```suggestion
ECHO %ESC%[45mhxRaylib Build%ESC%[0m
```
Don't put your name here imo @ForeignSasquatch

**Review by logo4poop — COMMENTED** [review]
Here are my suggestions to this PR. I don't think putting your name in the thanks section is appropriate. However that is up to @ForeignSasquatch.

**Review by ForeignSasquatch — COMMENTED** [review]
I dont think i will not merge this

**Comment by polybiusproxy** [comment]
> I dont think i will not merge this

ok, that's up to you!
i will just fix the suggested things that @logo4poop requested

**Comment by polybiusproxy** [comment]
ok

**Comment by ForeignSasquatch** [comment]
sorry im doing a full redesign so i have to close this

**Pull request closed without merging**